yarn.lock

```diff
@@ -743,6 +743,41 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@endo/env-options@^0.1.4":
+  version "0.1.4"
+  resolved "https://registry.yarnpkg.com/@endo/env-options/-/env-options-0.1.4.tgz#e516bc3864f00b154944e444fb8996a9a0c23a45"
+  integrity sha512-Ol8ct0aW8VK1ZaqntnUJfrYT59P6Xn36XPbHzkqQhsYkpudKDn5ILYEwGmSO/Ff+XJjv/pReNI0lhOyyrDa9mg==
```

Comment:
This is suspicious. It suggests that something has a `package.json` that is pinned to a version of this package that is not in the workspace.

I would like to try this alternative and see whether `package.json`s change or `yarn.lock` changes.

```console
> ./scripts/get-versions.sh . | ./scripts/set-versions.sh
> yarn
```

Reply:
Looks the same after starting over from a fresh master and then running those two commands.